README.rst

```diff
@@ -36,9 +36,14 @@ Kinto is a minimalist JSON storage service with synchronisation and sharing abil
 * `Issue tracker <https://github.com/Kinto/kinto/issues>`_
 * `Contributing <https://kinto.readthedocs.io/en/latest/community.html#how-to-contribute>`_
 * `Try our daily flushed instance at: https://kinto.dev.mozaws.net/v1/ <https://kinto.dev.mozaws.net/v1/>`_
+* `Docker Hub <https://hub.docker.com/r/kinto/kinto-server>`_
 
 Requirements
 ------------
 
 * **Python**: 3.6+
-* **Backends**: In-memory (development), Postgresql 9.5+ (production)
+* **Backends**: In-memory (development), PostgreSQL 9.5+ (production)
+
+Docker
+------
+ docker pull kinto/kinto-server
```

Comment:
I think the link to Docker Hub is OK, but I don't know if we need this here. It's already the third option shown on https://kinto.readthedocs.io/en/latest/tutorials/install.html, which you can get to from clicking the link to "Online documentation" above. What do you think?